app-emulation/qemu/qemu-2.11.0.recipe

```diff
@@ -7,12 +7,12 @@ achieves very good performance."
 HOMEPAGE="http://wiki.qemu.org/"
 COPYRIGHT="2003-2017 Fabrice Bellard"
 LICENSE="GNU GPL v2"
```

Comment:
There might be [more licenses and copyright holders][0]. Thanks Debian folks for sorting them out.

[0]: http://metadata.ftp-master.debian.org/changelogs/main/q/qemu/qemu_2.11+dfsg-1_copyright